woof-code/3builddistro

```diff
@@ -332,6 +332,9 @@ if [ -n "$PETBUILDS" ]; then
 		echo "Building without qemu-user-static"
 	fi
 
+	# set EXTRA_PROCESSING in _00build.conf to perform any processing required - e.g. for devx
+	[ $EXTRA_PROCESSING ] && . $EXTRA_PROCESSING
```

Comment:
This test _can_ (not saying it is) be ambiguous without a string comparison operator, `-n` for non-zero string length, `-z` for zero length.

Reply:
And there's a syntax error if it's undefined, you need to wrap "$EXTRA_PROCESSING" with quotes